src/openhmd.c

```diff
@@ -97,6 +98,18 @@ int OHMD_APIENTRY ohmd_ctx_probe(ohmd_context* ctx)
 	return ctx->list.num_devices;
 }
 
+const char* OHMD_APIENTRY ohmd_gets(ohmd_string_description type)
```

Comment:
I was about to do that, but I noted that the ohmd_list_gets() function also returns const char * instead of a return code.  Should both of these be changed or is it that strings are different, since they're statically allocated?